includes/object-cache.php

```diff
@@ -684,11 +684,17 @@ protected function connect_using_phpredis( $parameters ) {
                     'timeout' => $parameters['timeout'],
                     'read_timeout' => $parameters['read_timeout'],
                     'persistent' => $parameters['persistent'],
+                    'password' => null,
```

Comment:
We might not be able to just set it and have to set it to null or the actually password in the next condition to not pass to many parameters.

Reply:
I've modified the PR so that the password key is only set when it is known that version is sufficiently high.